VK_V1.0/VK_V1.0/AvailableGroups/AvailableGroupsTableViewCell.swift

```diff
@@ -0,0 +1,19 @@
+// AvailableGroupsTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячейка доступной группы
+final class AvailableGroupsTableViewCell: UITableViewCell {
+    // MARK: - Private IBOutlets
+
+    @IBOutlet private var avatarImageView: UIImageView!
+    @IBOutlet private var titleLabel: UILabel!
+
+    // MARK: - Public methods
+
+    func setup(data: Group) {
```

Comment:
```suggestion
    func setup(group: Group) {
```
Мне Женя менял так, что бы имена совпадали 
но это можно уточнить так как я тоже написала модель, кажется что так логичнее